pkg/upgrade/upgrade_test.go

```diff
@@ -178,3 +178,29 @@ func TestSkipForInvalidSemVer(t *testing.T) {
 	assert.NoError(t, cl.Get(context.Background(), nsn, persisted))
 	assert.Equal(t, "1.11.0", persisted.Status.Version)
 }
+
+func TestSkipUpgradeForVersionsGreaterThanLatest(t *testing.T) {
```

Comment:
Not sure how this is showing the upgrade was skipped?

Reply:
Well, is not really testing that, but the fact that I haven't touched the version, not assigned the latest one.

Reply:
Ah ok, thanks.